Archipelago.MultiClient.Net.Analyzers/Archipelago.MultiClient.Net.Analyzers.Test/ItemFlagsInSwitchTest.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.CodeAnalysis.Testing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VerifyCS = Archipelago.MultiClient.Net.Analyzers.Test.CSharpCodeFixVerifier<
+    Archipelago.MultiClient.Net.Analyzers.Analyzers.ItemFlagsInSwitchDiagnostics,
+    Archipelago.MultiClient.Net.Analyzers.Fixes.ItemFlagsInSwitchFixes>;
+
+namespace Archipelago.MultiClient.Net.Analyzers.Test
+{
+    [TestClass]
+    public class ItemFlagsInSwitchTest
```

Comment:
I think a test to confirm the correct generation of else-if's would be nice here